test/lib/util.h

```diff
@@ -24,4 +24,10 @@
 		}                                                           \
 	} while (0)
 
+static inline bool param_bool(const MunitParameter *params, const char *name)
```

Comment:
[Nit]: Maybe `param_to_bool`?

Reply:
I've changed it to `param_get_bool` which for me describes the intention better, does that work?